src/lib/dev-environment.js

```diff
@@ -0,0 +1,188 @@
+ * @flow
+ * @format
+ */
+
+/**
+ * External dependencies
+ */
+import debugLib from 'debug';
+import xdgBasedir from 'xdg-basedir';
+import os from 'os';
+import fs from 'fs';
+import ejs from 'ejs';
+import path from 'path';
+import Lando from 'lando/lib/lando';
+import landoUtils from 'lando/plugins/lando-core/lib/utils';
+
+/**
+ * Internal dependencies
+ */
+
+const debug = debugLib( '@automattic/vip:bin:dev-environment' );
+
+const containerImages = {
+	wordpress: {
+		image: 'wpvipdev/wordpress',
+		tag: '5.6',
+	},
+	jetpack: {
+		image: 'wpvipdev/jetpack',
+	},
+	muPlugins: {
+		image: 'wpvipdev/mu-plugins',
+		tag: 'auto',
+	},
+	clientCode: {
+		image: 'wpvipdev/skeleton',
+		tag: '181a17d9aedf7da73730d65ccef3d8dbf172a5c5',
+	},
+};
+
+export const defaults = {
+	environmentSlug: 'vip-local',
+	title: 'VIP Dev',
+	multisite: false,
+	phpVersion: '7.3',
+	jetpack: {
+		mode: 'inherit',
+	},
+	wordpress: {},
+	muPlugins: {},
+	clientCode: {},
+};
+
+[ 'wordpress', 'muPlugins', 'clientCode' ].forEach( type => {
+	defaults[ type ] = {
+		mode: 'image',
+		image: containerImages[ type ].image,
+		tag: containerImages[ type ].tag,
+	};
+} );
+
+const landoFileTemplatePath = 'assets/dev-environment.lando.template.yml.ejs';
+const landoFileName = '.lando.yml';
+
+export async function startEnvironment( slug, options ) {
+	debug( 'Will start an environment', slug, 'with options: ', options );
+
+	const instancePath = getEnvironmentPath( slug );
+
+	const alreadyExists = fs.existsSync( instancePath );
+
+	if ( alreadyExists ) {
+		const parameters = Object
+			.keys( options || {} )
+			.filter( key => key !== 'slug' );
+
+		if ( parameters && parameters.length ) {
+			throw new Error(
+				`The environment ${ slug } already exists and we can not change its configuration` +
+				` ( configuration parameters - ${ parameters.join( ', ' ) } found ).` +
+				' Destroy the environment first if you would like to recreate it.' );
+		}
+	} else {
+		const instanceData = generateInstanceData( slug, options );
+
+		debug( 'Instance data to create a new environment:', instanceData );
+
+		await prepareLandoEnv( instanceData, instancePath );
+	}
+
+	await landoStart( instancePath );
+}
+
+function getLandoConfig() {
+	const landoPath = path.join( __dirname, '..', '..', 'node_modules', 'lando' );
+
+	debug( `Getting lando config, using path '${ landoPath }' for plugins` );
+
+	return {
+		logLevelConsole: 'warn',
+		landoFile: '.lando.yml',
+		preLandoFiles: [ '.lando.base.yml', '.lando.dist.yml', '.lando.upstream.yml' ],
+		postLandoFiles: [ '.lando.local.yml' ],
+		pluginDirs: [
+			landoPath,
+			{
+				path: path.join( landoPath, 'integrations' ),
+				subdir: '.',
+			},
+		],
+	};
+}
+
+async function landoStart( instancePath ) {
+	debug( 'Will start lando app on path:', instancePath );
+
+	const lando = new Lando( getLandoConfig() );
+	await lando.bootstrap();
+
+	const app = lando.getApp( instancePath );
+	await app.init();
+
+	await app.start();
```

Comment:
It's so great that this works as a library! And we don't get the fancy messages, so it's an easy white labeling 😄